install/zowe-install.sh

```diff
@@ -104,24 +104,21 @@ echo "  Attempting to setup Zowe Explorer UI Plugins ... "
 # Configure the TLS certificates for the zLUX server
 . $INSTALL_DIR/scripts/zowe-zlux-configure-certificates.sh
 
-# install the liberty-server by expanding the /bin/atlas-pax file to ZOWE_ROOT_DIR/liberty-server/wlp/...
-. $INSTALL_DIR/scripts/zowe-install-liberty-server.sh
-
 # Add API Catalog application to zLUX - required before we issue ZLUX deploy.sh
 CATALOG_GATEWAY_URL=https://$ZOWE_EXPLORER_HOST:$ZOWE_APIM_GATEWAY_PORT/ui/v1/apicatalog
 . $INSTALL_DIR/scripts/zowe-install-iframe-plugin.sh $ZOWE_ROOT_DIR "org.zowe.api.catalog" "API Catalog" $CATALOG_GATEWAY_URL $INSTALL_DIR/files/assets/api-catalog.png
 
-echo "---- After expanding ZLUX.pax and Atlas.pax this is a directory listing of "$ZOWE_ROOT_DIR >> $LOG_FILE
+echo "---- After expanding ZLUX.pax this is a directory listing of "$ZOWE_ROOT_DIR >> $LOG_FILE
 ls $ZOWE_ROOT_DIR >> $LOG_FILE
 echo "-----"
-# run the atlasZluxInection script that copies folders into the zlux-app-server
```

Comment:
This comment isn't accurate anymore - I'm not even sure this file is needed, but outside the scope of this issue to review